components/style/properties/longhand/box.mako.rs

```diff
@@ -2122,9 +2131,11 @@ ${helpers.single_keyword("transform-style",
             if self.contains(STRICT) {
                 return dest.write_str("strict")
             }
-            if self.contains(CONTENT) {
-                return dest.write_str("content")
```

Comment:
Hmm, looking at this, I realize that the existing serialization (and storage for specified/computed values) is wrong.  The spec says that the computed value is the same as the specified value, which means that `contains: layout style paint` must serialize as `layout style paint` and not as `content`.

The current behavior will probably cause some Gecko test failures, so I think you'll need to fix that.  Maybe we can just make `STRICT` and `CONTENT` be separate bits, but when we parse those values, we also set the specific bits they correspond to.  (That's what Gecko does.)  Then `to_css` should work properly without any further changes.